core/app/[locale]/(default)/account/(tabs)/orders/page-data.ts

```diff
@@ -4,30 +4,14 @@ import { cache } from 'react';
 import { getSessionCustomerAccessToken } from '~/auth';
 import { client } from '~/client';
 import { PaginationFragment } from '~/client/fragments/pagination';
-import { graphql, ResultOf, VariablesOf } from '~/client/graphql';
-import { revalidate } from '~/client/revalidate-target';
+import { graphql, VariablesOf } from '~/client/graphql';
 
-import { OrderItemFragment } from './_components/product-snippet';
+import {
+  addProductAttributesToShippingConsignments,
+  OrderShipmentFragment,
+} from '../order/[slug]/page-data';
```

Comment:
Not exactly. We need this fragment because we use `ProductSnippet` for representing Order Item in 2 places here, in Orders page, as well as in specific Order page( slug page).